game/scripts/vscripts/abilities/twin/boss_twin_twin_empathy.lua

```diff
@@ -0,0 +1,7 @@
+LinkLuaModifier("modifier_boss_twin_twin_empathy_buff", "abilities/twin/modifier_boss_twin_twin_empathy.lua", LUA_MODIFIER_MOTION_NONE)
```

Comment:
The ability itself doesn't technically do anything though, it's not cast to apply the modifier, that's just done directly when spawning. Since the caster and parent need to refer to each of the two twins though I guess making it the intrinsic modifier of the ability wouldn't work. You might want to consider rewriting it to use casts of this ability to actually do the healing. Mainly because the modifier has an issue with updating the interval.